### ISSUE TITLE
get SIMD type should return the simd type that Milvus is running agaisnt in GetMetrics 

### ISSUE THREAD

**Issue by yanliang567** [milvus-io/milvus#8836]
#### Steps/Code to reproduce:
sys_info = handler._stub.GetMetrics(req, wait_for_ready=True, timeout=None)
nodes = json.loads(sys_info.response).get('nodes_info')
querynode.get('infos').get('system_configurations').get('simd_type')

#### Expected result:
get('simd_type') should return the simd type that Milvus is running against

#### Actual results:
it returns 'auto', which is the configuration of Milvus SIMD type

![image](https://user-images.githubusercontent.com/82361606/135206907-a365ab5f-23d9-413e-aa94-b7a48a80e4fa.png)

#### Environment:
- Milvus version(e.g. v2.0.0-RC2 or 8b23a93):
- Deployment mode(standalone or cluster):
- SDK version(e.g. pymilvus v2.0.0rc2):
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:

**Comment by DragonDriver** [comment]
working on this
check if we can get the simd type when the value is set to `auto`

**Comment by DragonDriver** [comment]
```cpp
Status
KnowhereConfig::SetSimdType(const SimdType simd_type) {
    if (simd_type == SimdType::AVX512) {
        faiss::faiss_use_avx512 = true;
        faiss::faiss_use_avx2 = false;
        faiss::faiss_use_sse = false;
    } else if (simd_type == SimdType::AVX2) {
        faiss::faiss_use_avx512 = false;
        faiss::faiss_use_avx2 = true;
        faiss::faiss_use_sse = false;
    } else if (simd_type == SimdType::SSE) {
        faiss::faiss_use_avx512 = false;
        faiss::faiss_use_avx2 = false;
        faiss::faiss_use_sse = true;
    } else {
        faiss::faiss_use_avx512 = true;
        faiss::faiss_use_avx2 = true;
        faiss::faiss_use_sse = true;
    }

    std::string cpu_flag;
    if (faiss::hook_init(cpu_flag)) {
        std::cout << "FAISS hook " << cpu_flag << std::endl;
        LOG_KNOWHERE_DEBUG_ << "FAISS hook " << cpu_flag;
        return Status::OK();
    }

    return Status(KNOWHERE_UNEXPECTED_ERROR, "FAISS hook fail, CPU not supported!");
}
```

**Comment by DragonDriver** [comment]
So when we set simd type to auto, in fact we set `avx12`, `avx2`, `sse` both. Then we called `faiss::hook_init`.
I will take a look at `faiss::hook_init`.

**Comment by DragonDriver** [comment]
fixed.
On my local pc, set to `auto` in `knowhere.yaml`, get `avx2` in metrics.
/assign @yanliang567

**Comment by yanliang567** [comment]
verified fix on master-20210930-615f72b
